pkg/apis/awsprovider/v1alpha1/awsmachineproviderconfig_types.go

```diff
@@ -72,8 +85,34 @@ type AWSMachineProviderSpec struct {
 	// KeyName is the name of the SSH key to install on the instance.
 	// +optional
 	KeyName string `json:"keyName,omitempty"`
+
+	// Placement specifies where to create the instance in AWS
+	Placement Placement `json:"placement"`
+
+	// LoadBalancers is the set of load balancers to which the new instance
+	// should be added once it is created.
+	LoadBalancers []LoadBalancerReference `json:"loadBalancers"`
 }
 
+// Placement indicates where to create the instance in AWS
+type Placement struct {
+	// Region is the region to use to create the instance
+	Region string `json:"region"`
```

Comment:
i think this field can be dropped, the cluster is scoped to a region already.

Reply:
Per our discussion, I think API proposal should probably be a separate PR.

Reply:
@derekwaynecarr https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/508